_includes/tutorials/optimize-producer-throughput/confluent/markup/dev/producer-perf-test-optimized-throughput.adoc

```diff
@@ -0,0 +1,19 @@
+Now rerun the Kafka producer performance test using optimized configuration values.
+This test will send the same number of records of the same size.
```

Comment:
I'm wondering if this is different than the first test, or not

Reply:
Tweaked a bit